armi/operators/tests/test_operators.py

```diff
@@ -76,5 +78,105 @@ def test_addInterfaceSubclassCollision(self):
         self.assertEqual(o.getInterface("Third"), interfaceC)
 
 
+class CyclesSettingsTests(unittest.TestCase):
+    """
+    Check that we can correctly access the various cycle settings from the operator.
+    """
+
+    detailedCyclesSettings = """
+metadata:
+  version: uncontrolled
+settings:
+  power: 1000000000.0
+  nCycles: 3
+  cycles:
+    - name: dog
```

Comment:
nbd but I kind of prefer like "physics testing" and "initial startup" or something slightly more analogous to what they'll really be called

Reply:
Done